framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -190,7 +191,7 @@ export const CreatableSelectDropdown = <T extends string = string>({
   }
 
   const combinedOptions = useMemo(
-    (): Option<T>[] => [ ...standardOptions, ...createdOptions ],
+    (): Option<T>[] => uniqBy(compose(toLower, trim, prop('value')), [ ...standardOptions, ...createdOptions ]),
```

Comment:
Are you sure about `toLower` and `trim`? If there existed an option called 'sweden' and the user then tried to create 'SWEDEN' it would not go through, similiar thing for trim